python_modules/libraries/dagster-dbt/dagster_dbt/asset_utils.py

```diff
@@ -225,11 +225,42 @@ def all_dbt_assets():
             # of them (dbt-related or otherwise)
             foo_and_downstream_selection = foo_selection.downstream()
 
+        Building an asset selection on a dbt assets definition with an existing selection:
+
+        .. code-block:: python
+
+            from dagster_dbt import dbt_assets, build_dbt_asset_selection
+
+            @dbt_assets(
+                manifest=...
+                select="bar+",
+            )
+            def all_dbt_assets():
+                ...
+
+            # Select the dbt assets that are in the intersection of having the tag "foo" and being
+            # in the existing selection "bar+".
+            foo_selection = build_dbt_asset_selection([dbt_assets], dbt_select="tag:foo")
```

Comment:
`[dbt_assets]` to either `[all_dbt_assets]` or `[bar_dbt_assets]` if you decide to rename.